ui/src/components/filter/permission.js

```diff
@@ -1,6 +1,10 @@
-export default (accessType, action, permissions) => {
-  if (permissions[accessType].includes(action)) {
-    return true;
-  }
-  return false;
-};
+// import { authorizer } from '../../authorizer';
+/**
+ *
+ * @param accessType
+ * @param action
+ * @param permissions
+ * @returns {boolean}
+ */
+// eslint-disable-next-line max-len
+export default (accessType, action, permissions) => !!permissions[accessType] && permissions[accessType].includes(action);
```

Comment:
Why double not (`!!`)?

Reply:
It "converts" a possible undefined in `permissions[accessType]` to false.